rllib/algorithms/algorithm_config.py

```diff
@@ -953,6 +954,16 @@ def validate(self) -> None:
                     f"config.framework({self.framework_str})!"
                 )
 
+        if (
```

Comment:
What if the simple optimizer is false but load mini batch is true and framework is tf. Do we support this?

Reply:
ok but this condition won't raise an error.

Reply:
What do you mean? It will raise an error if framework != torch!

Reply:
There is an and between `self.framework_str != "torch"` and `self.simple_optimizer`. For if tf is selected and self.simple_optimizer is False this condition will become false, and you won't raise an error?

Reply:
Oh my mistake, sorry! Thanks - I'm changing it